lib/config.js

```diff
@@ -23,6 +28,10 @@ module.exports = {
     return this.getNumberByPath(`rules["${ruleName}"][1][${ruleProperty}]`, defaultValue)
   },
 
+  getObjectPropertyBoolean(ruleName, ruleProperty, defaultValue) {
```

Comment:
This doesn't seem to be used anywhere in the rest of the PR.

Reply:
right, it was from some interim trial and errors.

Reply:
Actually now it is used.